tests/FSharp.Compiler.ComponentTests/TypeChecks/TypeExtensions/PropertyShadowing/E_CannotShadowFunctionPropertyWithExtensionMethod.fsx

```diff
@@ -0,0 +1,20 @@
+open System.Runtime.CompilerServices
+
+type Foo() = 
+    member val X : int -> int = (+) 1 with get,set
+
+[<Extension>]
+type FooExt =
+    [<Extension>]
+    static member X (f: Foo, i: int -> int) = f.X <- i; f
+
+let f = Foo()
+f.X 1
```

Comment:
I think it would be nice to add some Symbols tests to make sure the properties are available in tooling.

Reply:
@edgarfgp agreed, there are few snags to do so: 

I opened a PR https://github.com/dotnet/fsharp/pull/16258 due to few issues in the test infrastructure and had a brief chat with @psfinaki.

The crux is:
* right now we can't test "with preview" "without preview", despite I fixed something related to test options flags not given to FCS
* we don't have "baseline" infrastructure, to make such tests robust in long run: right now they just check if arbitrary values are present or not, just as a string (for this PR, we'd need at least the glyph to distinguish)
* the tests are only VS editor tests

For the later point, it is good to have those, but we should consider if putting tests with baselines on item list lower in the stack, given the bulk of it is implemented in FCS API.

This require some considerations, and for now, I've done a bunch of manual testing reviewing the .fsx scripts that are run under unit tests (very easy to try in debug VS as those are standalone files).

I'll resolve this point for now, but this is very true, we need better tests and infra, for the intellisense lists